gradle-plugin-api/src/main/java/wtf/emulator/TestReporter.java

```diff
@@ -0,0 +1,14 @@
+package wtf.emulator;
+
+public enum TestReporter {
+  /**
+   * Import result JUnit XML files to Develocity using the
+   * [ImportJUnitXmlReports task](https://docs.gradle.com/enterprise/gradle-plugin/api/com/gradle/develocity/agent/gradle/test/ImportJUnitXmlReports.html)
```

Comment:
javadoc does not support markdown, use html anchor tag instead?

```suggestion
   * <a href="https://docs.gradle.com/enterprise/gradle-plugin/api/com/gradle/develocity/agent/gradle/test/ImportJUnitXmlReports.html">ImportJUnitXmlReports task</a>
```